bin/znapzend

```diff
@@ -164,6 +165,23 @@ just look at this dataset's backup plan, but iterate into its children that
 might have any. Useful for quick backups of a pool whose root dataset has no
 I<znapzendzetup> configurations defined, but some trees under it do.
 
+See also B<--inherited> for interaction with that option.
+
+=item B<--inherited>
+
+when backing up a particular dataset with B<--runonce>=[I<dataset>], do not
+require that the provided dataset directly has a locally defined backup plan
+configuration (sanity check that it is not "received" via ZFS replication).
+With this option in place, allow also to quickly snapshot and/or replicate
+datasets of subtrees which inherit a backup plan from a dataset which has it
+in its attributes whose source is "local".
+
+At this time, the B<--inherited> is effectively exclusive vs. B<--recursive>
+mode (and preferred if both are specified) if the requested dataset has some
```

Comment:
if there is exclusivity I would raise an error if both are specified.

Reply:
Good point, I'll think how to re-word it. The idea was that de-facto one takes place, if both are set it depends on the explicitly requested dataset if it has a config, or in recursive mode the nearest dataset(s) under it having a config (likely local in this case... not sure if inherited is then even possible). So it would not recurse further, at least not into inherited-only configs,

One point to review is whether znapzend allows having another "local" backup plan config in a descendant dataset under one that has a config already.

Reply:
The documentation comments are hopefully addressed by "Revise doc...." commit now, thanks.

Reply:
having a separate backup plan underneath one that is marked as recursive is something @moetiker just talked to me yesterday ... at the moment this is not possible, but he would sure like it :)

Reply:
one of the reasons why we wanted recursive backups was to have consistant snapshots over a tree, so if there was a alternate plan somewhere down the tree that would make the snapshotting and espeacially the cleanup a bit more involved. although you might just ignore the 'local snapshot config' for the this case ... complaining about it if someone configured it.